provider/cluster_rosa_classic_resource.go

```diff
@@ -813,10 +860,14 @@ func (r *ClusterRosaClassicResource) getVersionList(logger logging.Logger, ctx c
 
 	return
 }
-func (r *ClusterRosaClassicResource) getVersions(logger logging.Logger, ctx context.Context) (versions []*cmv1.Version, err error) {
+func (r *ClusterRosaClassicResource) getVersions(logger logging.Logger, ctx context.Context, channelGroup string) (versions []*cmv1.Version, err error) {
 	page := 1
 	size := 100
-	filter := "enabled = 'true' AND rosa_enabled = 'true'"
+	filter := strings.Join([]string{
+		"enabled = 'true'",
+		"rosa_enabled = 'true'",
+		fmt.Sprintf("channel_group = '%s'", channelGroup),
```

Comment:
The channel group is now always provided in the query, even when on `stable`. This may clear up an issue I was having when getting started where versions that aren't actually available (in stable) can be seen.